fe-nc-news-project/src/Comments.jsx

```diff
@@ -1,34 +1,75 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { fetchCommentsByArticleId } from "./utils";
+import { fetchCommentsByArticleId, postComment } from "./utils";
 import Article from "./Article";
 
-export default function Comments() {
+export default function Comments({username}) {
   const [comments, setComments] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
+  const [commentText, setCommentText] = useState("");
+  const [posting, setPosting] = useState(false);
+  const [posted, setPosted] = useState(false);
   const { article_id } = useParams();
 
+  const id = article_id
+
   useEffect(() => {
     fetchCommentsByArticleId(article_id).then((result) => {
```

Comment:
It would be better to ensure that what is being returned from your fetchCommentsByArticleId is just the comments - at the moment, our function name suggests it fetches comments, but actually returns an object, with some data, with some comments on it.